### ISSUE TITLE
Accessiblity: Brisk vs onBoard

### ISSUE THREAD

**Issue by marcusulpius** [getsolus/brisk-menu#15]
[Accessbility: Brisk vs Onboard](https://bugs.launchpad.net/ubuntu/+source/brisk-menu/+bug/1793990)

When using a device without a keyboard such as a tablet on direct touchscreen, the Brisk Menu along with its top search field assumes/overrides the top most input layer position making it impossible to use the Accessiblity virtual keyboard Onboard to search for specific apps and programs.

This was originally opened under Ubuntu MATE then moved to Ubuntu proper. Any thoughts?

Basically this is killing Brisk use on a tablet without a keyboard.
